app/javascript/dashboard/routes/dashboard/notifications/components/NotificationPanelItem.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+  <div>
+    <woot-button
+      size="expanded"
+      color-scheme="secondary"
+      variant="link"
+      @click="() => onClickOpenNotification(notificationItem)"
```

Comment:
You don't need to pass the notificationItem here, since its already available

Reply:
I forgot it. Updated it now